src/screens/FeedScreen/Header.js

```diff
@@ -177,10 +226,35 @@ const _renderProfileNormal = ({
   height,
   source,
   headerStyle,
-  onHeaderOptionClicked = () => {},
-  hideThreeDot,
-  isPostDetail
+  isPostDetail,
+  isFollow = false,
+  onPressFollUnFoll = () => {},
+  onDeletePost = () => {},
+  isShowDelete = false,
+  isSelf = false
 }) => {
+  const refSheet = React.useRef();
+  const dataSheet = [
+    {
+      id: 1,
+      name: 'Share link',
+      icon: <ShareAndroidIcon />,
+      onPress: () => ShareUtils.shareUserLink(username)
+    }
+  ];
+
+  if (isShowDelete) {
+    dataSheet.push({
+      id: 2,
+      name: 'Delete post',
+      icon: <TrashRed />,
+      onPress: () => {
+        refSheet.current.close();
+        onDeletePost();
+      },
+      style: {color: colors.red}
+    });
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New props have been added to the component to support the new functionalities. It's important to ensure that these props are used correctly throughout the component and that they have appropriate default values if not provided.
```diff
+   isPostDetail,
+   isFollow = false,
+   onPressFollUnFoll = () => {},
+   onDeletePost = () => {},
+   isShowDelete = false,
+   isSelf = false
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->